src/api/server_mock.js

```diff
@@ -0,0 +1,36 @@
+import { createServer, Model, RestSerializer } from "miragejs";
```

Comment:
if not needed, kindly remove this unwanted file.

Reply:
Thanks for pointing out